source/users/custom-metadata.txt

```diff
@@ -589,129 +609,142 @@ Configure Authentication Provider Metadata
-                      "metadata_fields": [
-                        { "required": false, "name": "name" },
-                        { "required": true, "name": "first_name" },
-                        { "required": true, "name": "last_name" },
-                        { "required": false, "name": "picture" },
-                        { "required": false, "name": "gender" },
-                        { "required": false, "name": "birthday" },
-                        { "required": false, "name": "min_age" },
-                        { "required": false, "name": "max_age" },
-                        { "required": false, "name": "email" }
-                      ],
-                      "disabled": false
-                    }'
-
-            .. tab:: Custom JWT
-               :tabid: custom-jwt
-
-               .. code-block:: shell
-                  :emphasize-lines: 8-14
-
-                  curl --request PATCH 'https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/auth_providers/{providerId}' \
-                    --header 'Authorization:  Bearer <access_token>' \
-                    --header 'Content-Type: application/json' \
-                    --data '{
-                      "_id": "<Provider ID>",
-	                   "name": "custom-token",
-	                   "type": "custom-token",
-	                   "metadata_fields": [
-	                   	{
-	                   		"required": true,
-	                   		"name": "jwt.field.path",
-	                   		"field_name": "metadataFieldName"
-	                   	}
-	                   ],
-	                   "config": {
-	                   	"audience": [],
-	                   	"requireAnyAudience": false,
-	                   	"signingAlgorithm": "HS256",
-	                   	"useJWKURI": false
-	                   },
-	                   "secret_config": {
-	                   	"signingKeys": [
-	                   		"<JWT Signing Key>"
-	                   	]
-	                   },
-	                   "disabled": true
-                    }'
-
-         If you successfully configure the provider's metadata
-         fields, App Services returns a ``204`` response.
+                      "disabled": true
+                      }'
+
+          If you successfully configure the provider's metadata
+          fields, App Services returns a ``204`` response.
+
+
+
+.. note:: Tips for Users with Multiple Linked Authentication Providers  
 
+  - To ensure the most up-to-date metadata, users should re-authenticate after 
+    switching to a different authentication provider. Failure to do so may result 
+    in outdated metadata reflected in the :guilabel:`Users` table in 
+    the UI's :guilabel:`App Users` page and for requests using authentication providers.
+  - If a user switches between authentication providers, it may take up to 30 minutes 
+    to propagate metadata to the :guilabel:`Users` table. The request is always guaranteed to have the 
```

Comment:
after the comment above, can we just actually remove the "Users table" bit? Since the above bullet has both "Users Table" and "requests using authentication methods", I don't want this next bullet to only mention one of those (when it does apply to both but we're not including it). 
```suggestion
  - If a user switches between authentication providers, it may take up to 30 minutes 
   for metadata to propagate. The request is always guaranteed to have the 
```